test/e2e/framework/k8s/port_forward.go

```diff
@@ -0,0 +1,33 @@
+package k8s
+
+import (
+	"context"
+	"fmt"
+	"github.com/onsi/ginkgo"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/portforward"
+	"k8s.io/client-go/transport/spdy"
+	"net/http"
+)
+
+// NewPortForwarder return a new port forwarder
```

Comment:
Can you expand on this comment explaining how this is used by operator.

Reply:
e.g. To test whether pod(node-0-xxxx) can access service-1 from our code(which runs on mac or circleCI):

first, we setup a port-forward like localhost:8899:node-0-xxxx:8899. (httpProxy runs on 8899)
then we do curl -x "http://localhost:8899" "http://service-1:<service-port>/path-0"

Reply:
I think what Kiran meant is to add a little of the explanation to the comment above--which I think is a good idea.